### PR TITLE
#54 - Added DockerSlice with Version Check

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #54 
Added base Slice for Docker Registry with simple Version Check endpoint.

### CODE REVIEW

File: src/main/java/com/artipie/docker/http/DockerSlice.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ *
+ * @since 0.1
+ */
+public final class DockerSlice implements Slice {
+    @Override
+    public Response response(
+        final String line,
+        final Iterable<Map.Entry<String, String>> headers,
+        final Publisher<ByteBuffer> body) {
+        return new SliceRoute(
```
Comment (g4s8) [inline src/main/java/com/artipie/docker/http/DockerSlice.java]: @olegmoz can we put it to constructor like in `files-adapter`? https://github.com/artipie/files-adapter/blob/master/src/main/java/com/artipie/files/FilesSlice.java#L50-L56

File: src/test/java/com/artipie/docker/http/DockerSliceVersionCheckTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ *
+ * @since 0.1
+ */
+class DockerSliceVersionCheckTest {
+
+    @Test
+    void shouldRespondOkToVersionCheck() {
+        final DockerSlice slice = new DockerSlice();
+        final Response response = slice.response(
+            "GET /v2/",
```
Comment (g4s8) [inline src/test/java/com/artipie/docker/http/DockerSliceVersionCheckTest.java]: @olegmoz let's use `RequestLine` from `com.artipie.http.rq`: `new RequestLine(...).toString()`

**Comment by 0crat** [comment]
This pull request [#60](https://github.com/artipie/docker-adapter/issues/60) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/65d81947-4a84-4745-829f-6b760bdf5d10); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by olegmoz** [comment]
@g4s8 made changes by your comments, please review

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @g4s8[/z](https://www.0crat.com/u/g4s8)[/z](https://www.0crat.com/u/g4s8) didn't make CR comments

**Comment by 0crat** [comment]
Job was finished in 5 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Pull request merged**